noxfile.py

```diff
@@ -1,6 +1,6 @@
 import nox
 
 
-@nox.session(python=["3.6", "3.7", "3.8", "3.9", "3.10"])
+@nox.session(python=["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"])
```

Comment:
The pipelines fail, but at least the Python 3.10 and 3.11 succeed on my computer. I couldn't find anything in the [nox documentation](https://nox.thea.codes/en/stable/usage.html) about installing different Python interpreters. I tried installing `pyenv` but got the following error:

```
      ############################ NOTE ############################
      We are sorry, but this package is not installable with pip.
      
      Please read the installation instructions at:
      
      https://github.com/pyenv/pyenv#installation
      ##############################################################
```

Reading the link it seems it only supports the Homebrew package manager :unamused:

Reply:
Shouldn't you just add it here? https://github.com/rtts/djhtml/blob/main/.github/workflows/main.yml#L21 ?

Reply:
It didn't, but I managed to fix the pipelines anyway :partying_face: